passkey-authenticator/src/authenticator/get_assertion.rs

```diff
@@ -69,13 +72,12 @@ where
         // 7. Collect user consent if required. This step MUST happen before the following steps due
         //    to privacy reasons (i.e., authenticator cannot disclose existence of a credential
         //    until the user interacted with the device):
-        let flags = self.check_user(&input.options).await?;
+        let flags = self
+            .check_user(&input.options, maybe_credential.clone().ok())
```

Comment:
This can be changed like this, and then you don't need to add all the Clone derives to the error types
```suggestion
            .check_user(&input.options, maybe_credential.as_ref().ok().cloned())
```

Reply:
Thanks, nice tip! Applied!